packages/components/psammead-media-player/src/Guidance/index.jsx

```diff
@@ -17,13 +17,23 @@ const GuidanceWrapper = styled.div`
   height: 100%;
   position: absolute;
   background-color: ${GUIDANCE_BACKGROUND};
+  border: 1px solid transparent;
   color: ${C_WHITE};
+  @media screen and (-ms-high-contrast: active) {
+    background-color: transparent;
+  }
+`;
 
+const Content = styled.div`
+  display: flex;
+  align-items: center;
   padding: ${GEL_SPACING};
-
+  border-bottom: 1px solid transparent;
+  @media screen and (-ms-high-contrast: active) {
+    background-color: window;
+  }
   @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
```

Comment:
It looks like this `@media` block is missing a closing brace.

Reply:
I pushed a change that's already fixed this👍